system_tasks.yml

```diff
@@ -15,6 +15,26 @@
     - python-dev
     - gdal-bin
     - unzip
+    - python-apt
+
+# By default, Piecewise uses EFF's LetsEncrypt CA for obtaining SSL certificates.
+# If you have obtained SSL certificates from another source, you should comment out the 
+# lines below, ending at the line starting with: `shell: "certbot ...` and then edit 
+# the Nginx default site to point at your certificates: /conf/etc/nginx/sites-available/default
+- apt_repository:
+    repo: deb http://ftp.debian.org/debian jessie-backports main
+    state: present
+- name: Update apt database to get jessie-backports
```

Comment:
You're right. I can add the backports repo first and then a second apt update won't be needed. Will revise